docs/instance_groups.md

```diff
@@ -200,8 +203,7 @@ If you decide you don't need an InstanceGroup any more, you delete it using: `ko
 
 Example: `kops delete ig morenodes`
 
-No rolling-update is needed (and note this is not currently graceful, so there may be interruptions to
-workloads where the pods are running on those nodes).
+No `kops update cluster` nor `kops rolling-update` is needed, so **be careful** when deleting an instance group, your nodes will be deleted automatically (and note this is not currently graceful, so there may be interruptions to workloads where the pods are running on those nodes).
```

Comment:
Typo `no`

Reply:
Also, I would turn the content of the () to be a fresh sentence to make it easier to read.

Reply:
@mikesplain Could you explain the typo? I don't get it

Reply:
Nevermind! My mistake @klaus993!

Reply:
OK! Great! About the thing between parenthesis, these days I will maybe submit another PR to leave it all clear and understandable :)